modular/manager/bucket_migrate_scheduler.go

```diff
@@ -275,15 +316,15 @@ func (plan *BucketMigrateExecutePlan) startMigrateSchedule() {
 				}
 				log.Debugw("success to push migrate gvg task to queue", "migrateGVGUnit", migrateGVGUnit, "migrateGVGTask", migrateGVGTask)
 
-				// Update database: migrateStatus to migrating
-				migrateGVGUnit.MigrateStatus = Migrating
-
 				// update migrateStatus
 				err = plan.manager.baseApp.GfSpDB().UpdateMigrateGVGUnitStatus(migrateGVGUnit.Key(), int(migrateGVGUnit.MigrateStatus))
 				if err != nil {
 					log.Errorw("failed to update migrate gvg status", "gvg_unit", migrateGVGUnit, "error", err)
 					return
 				}
+
+				// Update database: migrateStatus to migrating
+				migrateGVGUnit.MigrateStatus = Migrating
```

Comment:
Why adjust the code position?

Reply:
to ensure the data consistence between DB and in-memory.